spec/forms/developer_search_form_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe DevelopersSearchForm do
+  describe '#search' do
+
+    context 'with no param' do
+      it 'return all developers' do
```

Comment:
`return` -> `returns`
Read through the `rspec` part in the following page again.
https://github.com/jp-ryuji/setup_rails/pull/5